docs/getting-started/index.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Getting Started
+layout: doc
+---
+
+# Getting Started with ESLint
+
+ESLint is a tool for identifying and reporting on patterns found in ECMAScript/JavaScript code, with the goal of making code more consistent and avoiding bugs. In many ways, it is similar to JSLint and JSHint with a few exceptions:
+
+* ESLint uses [Espree](https://github.com/eslint/espree) for JavaScript parsing.
+* ESLint uses an AST to evaluate patterns in code.
+* ESLint is completely pluggable, every single rule is a plugin and you can add more at runtime.
+
+## Installation
+
+You can install ESLint using npm:
+
+    npm install -g eslint
```

Comment:
I think this should be expended. Explaining local vs. global install, having ESLint as a dependency of your project, etc.